client/views/actions/kick.tpl

```diff
@@ -1,6 +1,6 @@
 {{> ../user_name nick=from}}
 has kicked
-{{> ../user_name nick=target}}
+{{> ../user_name nick=target mode=""}}
```

Comment:
This is restoring previous behaviour. Ideally we would show the real mode, yes. But I don't want to touch that code as I have another PR open that touches same code already.